proposals/lip-0051.md

```diff
@@ -1526,7 +1520,7 @@ def getChainID(tokenID: TokenID) -> ChainID:
 def isNativeToken(tokenID: TokenID) -> bool:
     return getChainID(tokenID) == OWN_CHAIN_ID
 ```
-
+d
 #### userSubstoreExists
```

Comment:
I was wandering why this function is not used in `mint`, `initializeUserAccount`, and `transfer` functions, where there are checks of `if userStore(address, tokenID) does not exist` kind are used (and also in Execution of Token Transfer command and Cross-chain Token Transfer Messages). Is it the same, or is using `userSubstoreExists` is in any way better, since it is already defined.